geometry_msgs/Pose2D[] pose_2d

geometry_msgs/Pose[] pose

geometry_msgs/Twist[] velocity_command

geometry_msgs/Wrench[] force_command

